third_party/terraform/website/docs/r/bigquery_table.html.markdown

```diff
@@ -41,6 +41,33 @@ resource "google_bigquery_table" "default" {
     env = "default"
   }
 
+  field {
+    name        = "permalink"
+    type        = "STRING"
+    mode        = "NULLABLE"
+    description = "The Permalink"
+  }
+
+  field {
+    name        = "state"
+    type        = "STRING"
+    mode        = "NULLABLE"
+    description = "State where the head office is located"
+  }
+}
+
+resource "google_bigquery_table" "default_schema_implementation" {
```

Comment:
Instead of including both `schema` + `field` examples as resources in the same config, mind adding a separate example config?